src/server/requirements.ts

```diff
@@ -16,12 +17,6 @@ export interface RequirementsData {
   java_version: number;
 }
 
-interface ErrorData {
-  message: string;
-  label: string;
-  openUrl: Uri;
-  replaceClose: boolean;
-}
 /**
  * Resolves the requirements needed to run the extension.
  * Returns a promise that will resolve to a RequirementsData if
```

Comment:
Just below here it mentions ErrorData. Can this just be removed now ?

Reply:
ErrorData wasn't being used to annotate anything, so I annotated the object that this was supposed to be used for. However, we should remove a lot of this code at some point, since we never do anything with the ErrorData object we make.

Reply:
See https://github.com/redhat-developer/vscode-xml/issues/761